integrations/cassandra/src/cassandra_test.go

```diff
@@ -75,7 +68,7 @@ func TestPopulateInventory(t *testing.T) {
 		"key_6":                 map[string]interface{}{"otherImportantPassword": "(omitted value)"},
 	}
 
-	if !reflect.DeepEqual(inventory, expected) {
-		t.Errorf("Expected: %v. Actual: %v", expected, inventory)
+	if !reflect.DeepEqual(i.Items(), expectedItems) {
```

Comment:
That's just a suggestion. Maybe we could get rid of `reflect` if we already use `assert`?

Reply:
Indeed 👍